layouts/partials/partnerships.html

```diff
@@ -3,16 +3,9 @@
 {{ if $data.partnerships.partnerships.enable }}
 {{ with $data.partnerships.partnerships }}
 <div>
-    <div class="mt-9 text-center sm:mt-11 lg:mt-14">
-        <h3 class="font-bold text-xl sm:text-2xl lg:text-3xl">
-            <b class="text-[#1D65A6]">{{ with .title }} {{ index (split . " ") 0 | safeHTML }} {{ end }}</b>
-            {{ with .title }} {{ after (len (index (split . " ") 0)) . | safeHTML }} {{ end }}
-        </h3>
-      </div>
-    
-      <ul class="flex mt-10 mx-auto justify-center space-x-20">
+      <ul class="sm:flex mt-10 mx-auto justify-center sm:space-x-24">
```

Comment:
I checked the changes on smaller screens and noticed that the Google logo's width shrunk a bit, so I made some minor changes and will explain them below.

`sm:` was added before `flex` and `space-x-24` so that this style only applies to screens with a min. width of 640px.
`space-x-20` increased to `space-x-24` to provide a little more space between the text on smaller screens since some text was added.

I didn't get a chance to mention making changes depending on the screen size. During the next talk, I'll make sure that is discussed. There is some language about this in the documentation.